tests/slicing/test-runner.py

```diff
@@ -190,6 +206,7 @@ def run_test(test, bccode, optafter, linkafter, args):
     execute(bccode, test.expectedoutput)
 
     for f in toremove:
+        from os import unlink
```

Comment:
The import in loop is not quite nice. I would prefer to factor out this code into a function, if you do not want to import `unlink` globally.

Reply:
Thanks! `unlink` is now imported globally.